supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc

```diff
@@ -42,39 +42,26 @@ Always use "Administration Portal", including the capital P. When other departme
 ==== image:images/yes.png[yes] Data Warehouse (noun)
 *Description*: The Manager includes a comprehensive management history database, which any application can use to extract a range of information at the data center, cluster, and host levels. Installing Data Warehouse creates the `ovirt_engine_history` database, to which the Manager is configured to log information for reporting purposes.
 
-Data Warehouse is optional in Red Hat Enterprise Virtualization (3.x). It is mandatory in Red Hat Virtualization (4.x).
+Data Warehouse is mandatory in Red Hat Virtualization.
 
 Always spell out in full and capitalize as shown here, unless part of a command. Use "Data Warehouse", not "the Data Warehouse", unless referring to "the Data Warehouse package", "the Data Warehouse service", etc.
 
 *Use it*: yes
 
 *Incorrect forms*: DWH, data warehouse, Dataware House
 
-*See also*:
-
-[discrete]
-[[details-pane]]
-==== image:images/yes.png[yes] details pane (noun)
-*Description*: The details pane shows detailed information about a selected item in the results list. If no items are selected, this pane is hidden. If multiple items are selected, the details pane displays information on the first selected item only. For a visual example, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.1/html-single/introduction_to_the_administration_portal/#Graphical_User_Interface_elements[Graphical User Interface Elements] in the _Introduction to the Administration Portal_. The screen shot applies to Red Hat Virtualization 4.1 and earlier versions (including 3.x).
-
-Tabs in the details pane are always referred to as, for example, "the General tab in the details pane". This terminology only applies to Red Hat Virtualization 4.1 and earlier. In 4.2 and later, use "details view".
-
-*Use it*: yes
-
-*Incorrect forms*:
-
-*See also*: xref:results-list[results list], xref:details-view[details view]
+*See also*: xref:monitoring_portal[Monitoring Portal]
 
 [discrete]
 [[details-view]]
 ==== image:images/yes.png[yes] details view (noun)
-*Description*: The details view shows detailed information about a selected item. This terminology only applies to Red Hat Virtualization 4.2 and later, where this information is no longer in a pane, but on a separate page accessed by clicking the name of the item. In 4.1 and earlier, use "details pane".
+*Description*: The details view shows detailed information about a selected item.
```

Comment:
I think using the word "shows" might not be recommended. I think 'displayed' might be a good alternative (for translation reasons). WDYT?

```*Description*: The details view displays detailed information about a selected item.
```